hack/run.sh

```diff
@@ -3,23 +3,34 @@
 set -euo pipefail
 
 if [[ -z $BOT_TOKEN ]]; then echo "BOT_TOKEN var must be set"; exit 1; fi
-if [[ -z $BOT_APP_TOKEN ]]; then echo "BOT_APP_TOKEN var must be set"; exit 1; fi
+if [[ -z $BOT_SIGNING_SECRET ]]; then echo "BOT_SIGNING_SECRET var must be set"; exit 1; fi
+
+work_dir=$(readlink -f $(dirname $0)/..)
 
 tmp_dir=$(mktemp -d)
-tmp_kk=$tmp_dir/build.kubeconfig
+tmp_kube=$tmp_dir/kubeconfigs
+mkdir $tmp_kube
+tmp_boskos=$tmp_dir/boskos
 trap 'rm -rf $tmp_dir' EXIT
 
-kubectl config view >$tmp_kk
-kubectl --kubeconfig=$tmp_kk config use-context app.ci
+cd $tmp_dir
+oc --context app.ci -n ci get secrets ci-chat-bot-kubeconfigs -o json > ci-chat-bot-kubeconfigs.json
+for key in `cat ci-chat-bot-kubeconfigs.json | jq -r '.data | keys[]'`
+do
+  cat ci-chat-bot-kubeconfigs.json | jq -r ".data[\"$key\"]" | base64 -d > ${tmp_kube}/${key}
+done
```

Comment:
This can all be done auto-magically with: `oc --context app.ci -n ci extract secrets/ci-chat-bot-kubeconfigs --to=${tmp_kube} --confirm